src/server/index.ts

```diff
@@ -27,6 +29,28 @@ export default async function (options: Options): Promise<void> {
     await startWebdriverServer(config, options);
   }
 
+  if (cluster.isPrimary) {
+    if (config.storybookAutorunCmd) {
+      logger().info(
+        `Storybook should be started via \`${config.storybookAutorunCmd}\` and be accessible at ${config.storybookUrl}`,
+      );
+      logger().info('Waiting Storybook...');
+      await tryAutorunStorybook(config);
+    } else {
+      logger().info(`Storybook should be started and be accessible at ${config.storybookUrl}`);
```

Comment:
Should we use resolved storybook url if we have `resolveStorybookUrl` in the config?

Reply:
As I understand it, this URL is [dynamically generated](https://github.com/skbkontur/retail-ui/blob/next/packages/react-ui/.creevey/config.js#L25) and is useful for Selenium Grid. I made it have higher priority than `storybookUrl` if it is set